components/page.tsx

```diff
@@ -213,7 +214,7 @@ function PageContent() {
       name: "Maya Rainer",
       title: "Chief Destroyer of Technical Debt",
       url: "https://www.twitch.tv/mayarainer",
-      icon: <Trash2 size={28} className="text-current" />,
```

Comment:
remove the import for `Trash2` as it doesn't seem used in this file anymore please